source/ShopPanel.cpp

```diff
@@ -265,54 +266,136 @@ void ShopPanel::DrawSidebar()
 	if(playerShip)
 	{
 		point.Y() += SHIP_SIZE / 2;
-		point.X() = Screen::Right() - SIDE_WIDTH / 2;
+		point.X() = Screen::Right() - SHIP_SIDE_WIDTH / 2;
 		DrawShip(*playerShip, point, true);
 		
-		Point offset(SIDE_WIDTH / -2, SHIP_SIZE / 2);
+		Point offset(SHIP_SIDE_WIDTH / -2, SHIP_SIZE / 2);
 		sideDetailHeight = DrawPlayerShipInfo(point + offset);
 		point.Y() += sideDetailHeight + SHIP_SIZE / 2;
 	}
 	else if(player.Cargo().Size())
 	{
-		point.X() = Screen::Right() - SIDE_WIDTH + 10;
+		point.X() = Screen::Right() - SHIP_SIDE_WIDTH + 10;
 		font.Draw("cargo space:", point, medium);
 		
 		string space = Format::Number(player.Cargo().Free()) + " / " + Format::Number(player.Cargo().Size());
 		Point right(Screen::Right() - font.Width(space) - 10, point.Y());
 		font.Draw(space, right, bright);
 		point.Y() += 20.;
 	}
-	maxSideScroll = max(0., point.Y() + sideScroll - Screen::Bottom() + BUTTON_HEIGHT);
+	maxSideShipScroll = max(0., point.Y() + sideShipScroll - Screen::Bottom() + BUTTON_HEIGHT);
 	
 	PointerShader::Draw(Point(Screen::Right() - 10, Screen::Top() + 10),
-		Point(0., -1.), 10.f, 10.f, 5.f, Color(sideScroll > 0 ? .8f : .2f, 0.f));
+		Point(0., -1.), 10.f, 10.f, 5.f, Color(sideShipScroll > 0 ? .8f : .2f, 0.f));
 	PointerShader::Draw(Point(Screen::Right() - 10, Screen::Bottom() - 80),
-		Point(0., 1.), 10.f, 10.f, 5.f, Color(sideScroll < maxSideScroll ? .8f : .2f, 0.f));
+		Point(0., 1.), 10.f, 10.f, 5.f, Color(sideShipScroll < maxSideShipScroll ? .8f : .2f, 0.f));
+}
+
+
+
+void ShopPanel::DrawDetailsSidebar()
+{
+	const Font &font = FontSet::Get(14);
+	const Color &bright = *GameData::Colors().Get("bright");
+	sideDetailHeight = 0;
+
+	// Draw this string representing the selection, centered in the details side panel,
+	// It will either be the outfit name or the ship model.
+	string selectedItem = "Nothing Selected";
```

Comment:
Ideally there would be a single iteminfodisplay that you could pull these details from, but that's a larger refactor than needed here (one I have already started).

Reply:
Since you have already started working on this, can we presume this change request resolved?